README.md

```diff
@@ -28,57 +33,102 @@ Project DeepSpeech is an open source Speech-To-Text engine. It uses a model trai
 
-If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommended to install TensorFlow with GPU support. Training will likely be significantly quicker than using the CPU.
+```bash
+python util/tc.py destination/folder cpu
+```
 
-## Training a model
+This will download the native client files for the x86_64 architecture without CUDA support, and extract them into `destination/folder`. If you prefer building the binaries from source, see the [native_client README file](native_client/README.md). We also have binaries with CUDA enabled ("gpu") and for ARM7 ("arm").
 
-The central (Python) script is `DeepSpeech.py` in the project's root directory. For its list of command line options, you can call:
+### Recommendations
+
+If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommended to install TensorFlow with GPU support. Training will likely be significantly quicker than using the CPU. The [requirements.txt](requirements.txt) file installs the `tensorflow` package by default, which doesn't have GPU support enabled. To enable GPU support, you can do:
 
 ```bash
-$ ./DeepSpeech.py --help
+pip uninstall tensorflow
+pip install tensorflow-gpu
 ```
 
-To get the output of this in a slightly better-formatted way, you can also look up the option definitions top of `DeepSpeech.py`.
+### Training a model
 
-You'll need to download `native_client.tar.xz` or build the native client files yourself to get the custom TensorFlow OP needed for decoding the outputs of the neural network. You can use `util/tc.py` to download the files for your architecture:
+The central (Python) script is `DeepSpeech.py` in the project's root directory. For its list of command line options, you can call:
 
 ```bash
-python util/tc.py destination/folder cpu
+$ ./DeepSpeech.py --help
 ```
 
-This will download the native client files for the x86_64 architecture without CUDA support, and extract them into `destination/folder`. If you prefer building the binaries from source, see the [native_client README file](native_client/README.md). We also have binaries with CUDA enabled ("gpu") and for ARM7 ("arm").
+To get the output of this in a slightly better-formatted way, you can also look up the option definitions top of `DeepSpeech.py`.
 
 For executing pre-configured training scenarios, there is a collection of convenience scripts in the `bin` folder. Most of them are named after the corpora they are configured for. Keep in mind that the other speech corpora are *very large*, on the order of tens of gigabytes, and some aren't free. Downloading and preprocessing them can take a very long time, and training on them without a fast GPU (GTX 10 series recommended) takes even longer. If you experience GPU OOM errors while training, try reducing `batch_size`.
```

Comment:
With the amount of OOM problems we see. This seems like burying the lede. I'd guess we should try to make this more prominent.